src/bin/bitmaskd.rs

```diff
@@ -139,16 +147,17 @@ async fn main() -> Result<()> {
     pretty_env_logger::init();
 
     let app = Router::new()
+        /* TODO: ECDH bearer auth.
```

Comment:
Doubt, how will this work?

Reply:
What, ECDH bearer auth? Yes, we already do something similar with carbonado_node.

If you're asking if it's okay to comment out the node methods, yes, this is fine for now, since with this, RGB and Carbonado all work pretty much entirely clientside, with zero server calls except for Carbonado persistence and Electrum.

Reply:
Oh, I misread your question. It's been a long day. I'll show you how tomorrow.